test/property_model_eval/queryable_sheet.cpp

```diff
@@ -44,35 +43,41 @@ void queryable_sheet_t::begin_monitoring() {
         std::size_t i(iter->second);
         name_t& name(name_m[i]);
         sheet_m.monitor_enabled(name, NULL, NULL,
-                                boost::bind(adobe::assign(), _1, boost::ref(active_m[i])));
+                                std::bind(adobe::assign(), ph::_1, std::ref(active_m[i])));
 
         std::vector<name_t> v;
         v.push_back(name);
         sheet_m.monitor_enabled(
             dummy, &v[0], 1 + &v[0],
-            boost::bind(adobe::assign(), _1, boost::ref(priority_accessed_m[i])));
+            std::bind(adobe::assign(), ph::_1, std::ref(priority_accessed_m[i])));
 
-        sheet_m.monitor_value(name, boost::bind(adobe::assign(), _1, boost::ref(value_m[i])));
+        sheet_m.monitor_value(name, [&value_m_at_index_i = value_m[i]](any_regular_t val) {
+            adobe::assign{}(val, value_m_at_index_i);
+        });
 
         sheet_m.monitor_contributing(
-            name, dictionary_t(), boost::bind(adobe::assign(), _1, boost::ref(contributors_m[i])));
+            name, dictionary_t(), std::bind(adobe::assign(), ph::_1, std::ref(contributors_m[i])));
     }
 
     for (queryable_sheet_t::index_t::iterator iter = output_index_m.begin(),
                                               e = output_index_m.end();
          iter != e; ++iter) {
         std::size_t i(iter->second);
-        sheet_m.monitor_value(name_m[i], boost::bind(adobe::assign(), _1, boost::ref(value_m[i])));
+        sheet_m.monitor_value(name_m[i], [&value_m_at_index_i = value_m[i]](any_regular_t val) {
```

Comment:
`std::bind` is not working with type `any_regular_t` for `adobe::assign`.

Reply:
Do we have an understanding of why? Maybe it would be better to fix the underlying cause.